Makefile

```diff
@@ -1,5 +1,7 @@
+PKG_VER=$(shell grep rpiboot debian/changelog | head -n1 | sed 's/.*(\(.*\)).*/\1/g')
```

Comment:
Surely not `rpiboot`

Reply:
That's the current format of the changelog

Reply:
Oh sorry, for some reason I thought this was all for rpi-eeprom.